.github/workflows/lint.yml

```diff
@@ -21,26 +21,7 @@ jobs:
           ignore: "./**/CHANGELOG.md"
           args: "./**/*.md"
 
-      - name: restore lerna
-        uses: actions/cache@master # must use unreleased master to cache multiple paths
-        id: cache
-        with:
-          path: |
-            ./node_modules
-            ./package-lock.json
-            node_modules
-            detectors/node/*/node_modules
-            metapackages/*/node_modules
-            packages/*/node_modules
-            plugins/node/*/node_modules
-            plugins/web/*/node_modules
-            propagators/*/node_modules
-          key: ${{ runner.os }}-${{ hashFiles('**/package.json') }}
-
-      - name: Bootstrap
-        if: steps.cache.outputs.cache-hit != 'true'
-        run: |
-          npm install --only=dev --ignore-scripts
-          npx lerna bootstrap --no-ci --hoist --nohoist='zone.js' --nohoist='gts' --ignore-scripts -- --only=dev
+      # XXX consider 'npm install --only=dev' instead
```

Comment:
we should give it a try since it will improve in time and resources

Reply:
Nope, can't use this:

```
% npm install --only=dev
npm WARN invalid config only="dev" set in command line options
npm WARN invalid config Must be one of: null, prod, production
...
```

Same for `npm ci --only=dev`.  I don't know if the pre-existing usage of `npm install --only=dev` was hopeful usage, or if it did work in earlier versions of npm.